slices/sensible-utils.yaml

```
@@ -0,0 +1,26 @@
+package: sensible-utils
+
+essential:
+  - sensible-utils_copyright
+
+slices:
+  bins:
+    contents:
+      /usr/bin/select-editor:
+      /usr/bin/sensible-browser:
+      /usr/bin/sensible-editor:
+      /usr/bin/sensible-pager:
+      /usr/bin/sensible-terminal:
+
+  auxillary-bins:
```

```suggestion
  extras:
```

this is a name we already use in other pkgs (like python)

---

fixed!